book_vol_1.txt

```diff
@@ -412,7 +428,104 @@ Structure of Registers
         +----------------------------------+
                          |EXTENDED|   SP   |
                          +-----------------+
-                         31                0
+                         31       16       0

```

Comment:
I never mention R8-R15 during the book. So I didn't introduce them. Don't want to spam the reader.